jellyfin-api/src/main/kotlin/org/jellyfin/sdk/api/client/Response.kt

```diff
@@ -9,10 +9,15 @@ import kotlin.reflect.KProperty
  * @param status - See [HttpStatusCode]
  */
 public class Response<T>(
-	public val content: T,
+	public val content: T?,
 	public val status: Int,
 	public val headers: Map<String, List<String>>
 ) {
+	/**
+	 * Expect the [content] to be not-null.
+	 */
+	public val requireContent: T get() = content!!
```

Comment:
Don't know, should maybe be a function `requireContent()` or be called `requiredContent` or something.

Reply:
Doing it as a property makes it easier to use in my opinion. Easier to write `xx.requireContent.yy`. Although I can understand why you want it as function. I don't have a preference for one or the other so you call it.

Reply:
Maybe we don't need it at all? Because you can still use `by xx`.

```kotlin
val info = systemApi.getPublicInfo()

// a
val result by info

// b
val result = info.content!!

// c
val result = info.requireContent

// d
val result = info.requireContent()
```

Reply:
Both work fine, but with the current name it sounds more like a function. Properties do look neater in this case though, you're right - but then a different name should be used.

Reply:
> Maybe we don't need it at all? Because you can still use `by xx`.
> 
> ```kotlin
> val info = systemApi.getPublicInfo()
> 
> // a
> val result by info
> 
> // b
> val result = info.content!!
> 
> // c
> val result = info.requireContent
> 
> // d
> val result = info.requireContent()
> ```

Didn't see the updated comment yet, you're right - it isn't really necessary and can also be removed.

Reply:
Removed the requireContent prop and added a `requireNotNull` check in the `getValue` function to better message developers of a bad request,